app/javascript/react/utils/thresholdColors.ts

```diff
@@ -13,7 +13,7 @@ const COLORS_FOR_RANGES = (thresholdValues: Thresholds) => [
 
 const getColorForValue = (
   thresholdValues: Thresholds,
-  value: number | null
+  value: string | number | null
```

Comment:
why would value be a string in case of thresholds?

Reply:
For the new indoor points where there is no data yet, we receive the "no data" string as the average value, so we need to handle such a situation and display defaultColor.